microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -72,6 +72,11 @@ With this release, you can run {microshift-short} and your workloads with either
 ==== Resource footprint reduction for LVM storage driver now available
 With this release, the resource footprint of the LVM storage driver is significantly reduced, particularly in terms of memory consumption.
 
+[id="microshift-4-17-LVMS-volume-snapshot-deployment-optional_{context}"]
+==== Disabling and uninstalling LVMS CSI provider and CSI snapshot deployments now available
+With this release, {microshift-short} can now be configured to disable and uninstall the CSI provider or the CSI snapshot capabilities which can reduce the use of runtime resources such as RAM, CPU, and storage. See
+//xref:..microshift_storage/microshift-storage-plugin-overview.adoc#microshift-disabling-uninstalling-lvms-csi-snapshot_microshift-storage-plugin-overview[Disabling and uninstalling LVMS CSI provider and CSI snapshot deployments].
```

Comment:
Since this is commented out I can't check if the xref works.

Reply:
I will enable the link once the main PR is merged.